### PR TITLE
Add verbose options (--login and --interactive) to the argument parser

### PR DESCRIPTION
**Pull request by melund**

### CODE REVIEW

File: xonsh/main.py [hunk 1]
```diff
@@ -51,12 +51,12 @@ def path_argument(s):
...
                     required=False,
                     default=None)
-parser.add_argument('-i',
+parser.add_argument('-i', '--interactive',
                     help='force running in interactive mode',
                     dest='force_interactive',
                     action='store_true',
                     default=False)
-parser.add_argument('-l',
+parser.add_argument('-l', '--login',
```
Comment (scopatz) [inline xonsh/main.py]: Can you also add these to the argundoers below, please?

Comment (melund) [inline xonsh/main.py]: Sure. I didn't see them. What are they for?

Comment (scopatz) [inline xonsh/main.py]: Basically, they add these elements back in to the `args` or `argv` list.  This is needed if you are writing a command line script written in xonsh, ie `#!/usr/bin/env xonsh`.  The way argparse works is that it consumes these and thus they won't necessarily get passed down to the utility.  

Say that you were writing a command `test`, running `test -h` would give you xonsh's help NOT test's help. The arg undoer mitigate this.

File: xonsh/main.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def arg_undoers():
         '-h': (lambda args: setattr(args, 'help', False)),
         '-V': (lambda args: setattr(args, 'version', False)),
         '-c': (lambda args: setattr(args, 'command', None)),
-        '-i': (lambda args: setattr(args, 'force_interactive', Fals)),
+        '-i': (lambda args: setattr(args, 'force_interactive', False)),
```
Comment (melund) [inline xonsh/main.py]: @scopatz ... I also fixed this. I didn't think it was intended.

Comment (scopatz) [inline xonsh/main.py]: Oh, yeah! That was bad. Thanks :)

**Comment by scopatz** [comment]
Thanks a ton @melund!

**Pull request merged**